public/mcphelper/README.md

```
@@ -0,0 +1,564 @@
+| `plugin__with__dunders` | **undefined** — do not use |
+
+---
+
+## Known Limitations
+
+- **Not yet `go get`-able.** Until the Agents plugin cuts a release tag
+  with `public/mcphelper/` exported, consumers must point their `go.mod`
+  at a local checkout of the Agents plugin worktree via a `replace`
+  directive. Example:
+
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)